zarr/tests/test_n5.py

```diff
@@ -35,3 +39,14 @@ def test_partial_chunk_decode(chunk_shape: Tuple[int, ...]):
     chunk[subslices] = 1
     subchunk = np.ascontiguousarray(chunk[subslices])
     assert np.array_equal(codec_wrapped.decode(codec_wrapped.encode(subchunk)), chunk)
+
+
+def test_dtype_decode():
```

Comment:
This test will need the fsspec check:

```
@pytest.mark.skipif(have_fsspec is False, reason="needs fsspec")
```

Reply:
Thank you! i didn't know how to do it. i added it.

Reply:
Cheers!